packages/editor/package.json

```diff
@@ -64,6 +64,8 @@
 		"@wordpress/url": "file:../url",
 		"@wordpress/warning": "file:../warning",
 		"@wordpress/wordcount": "file:../wordcount",
+		"change-case": "^4.1.2",
+		"client-zip": "^2.4.4",
```

Comment:
Should we remove this one from the edit-site dependencies?

Reply:
Good catch, client-zip should be removed.